gitdm_client/README.md

```
@@ -0,0 +1,19 @@
+# GitDM Flutter Client
+
+این پوشه یک اسکلت ساده برای کلاینت Flutter جهت ارتباط با بک‌اند Django/DRF پروژه‌ی GitDM است.
+
+## اجرای اولیه
+```bash
+flutter pub get
+flutter run --dart-define=API_BASE_URL=http://127.0.0.1:8000
+```
+
+> برای اندروید، اطمینان حاصل کنید که مجوز اینترنت در `android/app/src/main/AndroidManifest.xml` اضافه شده باشد:
+> ```xml
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)